release-notes/formats.md

```diff
@@ -0,0 +1,101 @@
+
+Examples:
+
+- [.NET 9 Preview 1](./9.0/preview/preview1/README.md)
+- [.NET 9 Preview 6](./9.0/preview/preview6/README.md)
+
+These markdown files are primarily composed of dense feature descriptions. This content is used as source material for [What's New](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-9/overview) pages in official Microsoft docs, which are [often updated](https://github.com/dotnet/docs/pulls?q=is%3Apr+What%27s+New) on the same day as a preview release.
+
+Preview release notes are always in a `preview` folder. This approach was adopted so that preview releases do distract from stable releases once preview releases are no longer relevant.
+
+## Releases Index (json)
+
+The `releases-index.json` file is the entrypoint into all other JSON files. This file is relatively small, about a dozen lines per major version. The file includes support phase, latest version, and whether the latest version included CVE fixes.
+
+- File: [`releases-index.json`](./releases-index.json)
+- Schema: [`dotnet-releases-index.json`](https://json.schemastore.org/dotnet-releases-index.json)
+
+The following example demonstrates a version object from the file.
+
+```json
+{
+    "channel-version": "8.0",
+    "latest-release": "8.0.7",
+    "latest-release-date": "2024-07-09",
+    "security": true,
+    "latest-runtime": "8.0.7",
+    "latest-sdk": "8.0.303",
+    "product": ".NET",
+    "support-phase": "active",
+    "eol-date": "2026-11-10",
+    "release-type": "lts",
+    "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/8.0/releases.json",
+    "supported-os.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/8.0/supported-os.json"
+}
+```
+
+Each version object includes links to more detailed release-specific information.
+
+The links are to a blob storage location, not to GitHub. We chose the blog storage links as the default because we knew some users were using the JSON files from within Azure and we also better understood the reliability characteristics of Azure.
+
+`releases-index.json` and all the files it references are stored in blob storage and in GitHub. It is straightforward to programmatically transform the [blob storage links](https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json) to [`raw` GitHub links](https://raw.githubusercontent.com/dotnet/core/main/release-notes/releases-index.json) should that be desired.
+
+## Releases (json)
+
+The `releases.json` file contains expansive release information for the life of a major release. It includes much the same information as the Monthly and preview markdown release notes.
+
+Examples:
+
+- [8.0 `releases.json`](./8.0/releases.json)
+- [9.0 `releases.json`](./9.0/releases.json)
+
+This files can grow to be quite large. We are considering producing release-specific versions of this file, for example for `8.0/8.0.1/release.json`.
```

Comment:
Is there an issue to point to here?

Reply:
https://github.com/dotnet/core/issues/9417